network/crl_test.go

```diff
@@ -324,27 +321,27 @@ func TestDefaultCRLClientHTTP(t *testing.T) {
 	crlClient := NewDefaultCRLClient()
 
 	//
-	// Test with valid URI
+	// Test with valid URL
 	//
-	uri := fmt.Sprintf("http://%s/test_crl.pem", addr)
+	url := fmt.Sprintf("http://%s/test_crl.pem", addr)
 
-	fetchedCRL, err := crlClient.Fetch(uri)
+	fetchedCRL, err := crlClient.Fetch(url)
 	if err != nil {
-		t.Fatalf("Unexpected error during reading %s: %v\n", uri, err)
+		t.Fatalf("Unexpected error durlng reading %s: %v\n", url, err)
 	}
 
 	if !bytes.Equal(rawCRL, fetchedCRL) {
 		t.Fatal("CRLClient returned mismatched data\n")
 	}
 
 	//
-	// Test with invalid URI (i.e. when server returns 404 not found or any other error)
+	// Test with invalid URL (i.e. when server returns 404 not found or any other error)
 	//
-	uri = fmt.Sprintf("http://%s/wrong_filename.pem", addr)
+	url = fmt.Sprintf("http://%s/wrong_filename.pem", addr)
 
-	fetchedCRL, err = crlClient.Fetch(uri)
+	fetchedCRL, err = crlClient.Fetch(url)
 	if err == nil {
-		t.Fatal("Unexpected success during reading CRL from wrong URI\n")
+		t.Fatal("Unexpected success durlng reading CRL from wrong URL\n")
```

Comment:
You went a bit overboard with `s/uri/url/g` here and in some other log lines.

P.S. There's "word boundary" match, as in`/\buri\b/`.

Reply:
Didn't notice my regex replaced too much. Thanks, fixed those typos